homeassistant/components/miele/__init__.py

```diff
@@ -75,19 +80,25 @@ async def async_setup_entry(hass: HomeAssistant, entry: MieleConfigEntry) -> boo
         ) from err
 
     # Setup MieleAPI and coordinator for data fetch
-    api = MieleAPI(auth)
-    coordinator = MieleDataUpdateCoordinator(hass, entry, api)
+    entry.runtime_data = MieleRuntimeData  # type: ignore[assignment]
```

Comment:
You're now assigning the type, not an instance of the object

Reply:
I suggest this correction. Otherwise we have to shuffle things around in a way that makes the code less readable IMO.